### PR TITLE
feat: Add additional labels to containerdisks

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

Add additional labels to containerdisks which can carry information like an
appropriate instancetype or preference to use with the containerdisk.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

Merge after #74 

**Release note**:

```release-note
Provide labels with default instancetype and preference in containerdisks
```

### CODE REVIEW

**Comment by jcanocan** [comment]
Looks like `pull-containerdisks-test` is not happy. Can you please take a look?

**Comment by 0xFelix** [comment]
@jcanocan That is fixed by rebasing on #74 + https://github.com/kubevirt/project-infra/pull/3214 :)

**Comment by jcanocan** [comment]
> @jcanocan That is fixed by rebasing on #74 + [kubevirt/project-infra#3214](https://github.com/kubevirt/project-infra/pull/3214) :)

All right! Thanks!
/lgtm

File: cmd/medius/common/registry.go [hunk 1]
```diff
@@ -25,79 +26,137 @@ type Entry struct {
...
-		Artifact:   rhcosprerelease.New("latest"),
+		Artifact: rhcosprerelease.New(
+			"latest",
+			defaultLabels("u1.small", "rhel.9"),
+		),
 		UseForDocs: false,
 	},
 	{
-		Artifact:   centos.New("8.4"),
+		Artifact:   centos.New("8.4", nil),
```
[the remark this answers is not in the document]
Comment (0xFelix) [reply cmd/medius/common/registry.go:111]: We don't have a preference for Centos 8 (anymore).

**Comment by jcanocan** [comment]
Great! Thanks for the code and for answering my concerns.
/lgtm

**Pull request merged**